packages/tasit-action/src/TasitAction.test.js

```diff
@@ -199,11 +205,10 @@ describe("TasitAction.Contract", () => {
 
       await mineBlocks(provider, 15);
 
-      txSubscription.off("error");
-
       expect(confirmationFakeFn.callCount).to.equal(1);
       expect(errorFakeFn.called).to.be.false;
-      expect(txSubscription.subscribedEventNames()).to.be.empty;
```

Comment:
I was thinking that this expectation would move below the removal of the error event listener rather than being deleted

https://github.com/tasitlabs/TasitSDK/pull/102#discussion_r247598590

Reply:
Ah, ok. Sorry about that.